### PR TITLE
Document the prefix option

### PR DESCRIPTION
**Pull request by panda-madness**
This is sort of an important thing.

### CODE REVIEW

File: docs/configuration/database.md [hunk 1]
```diff
@@ -73,6 +73,16 @@ database:
...
 ```
 
+The default prefix for the database tables is `bolt_`. You can change this with the `prefix` option like so:
+```yaml
+database:
+  driver: mysql
+  username: bolt
+  password: password
+  databasename: bolt
+  prefix: yourapp_
```
Comment (rossriley) [inline docs/configuration/database.md]: the underscore isn't necessary, Bolt automatically adds this.

Comment (GawainLynch) [inline docs/configuration/database.md]: Oh, and 4 spaces is the default for indents too :smile:

Comment (panda-madness) [inline docs/configuration/database.md]: 2 spaces was how it was initially in the previous code blocks though.

**Comment by GawainLynch** [comment]
Quality work, thank you! :+1:

**Pull request merged**